spec/models/observation_spec.rb

```diff
@@ -2425,67 +2456,110 @@ def stub_api_response_for_observation( o )
     end
   end
 
-  describe "geom" do
-    it "should be set with coords" do
-      o = Observation.make!(:latitude => 1, :longitude => 1)
-      expect(o.geom).not_to be_blank
-    end
-  
-    it "should not be set without coords" do
-      o = Observation.make!
-      expect(o.geom).to be_blank
+  describe "#set_geom_from_latlon" do
```

Comment:
The coverage of the method is maintained below, however added this as each `it` block was inherently testing that the callback existed. Added this in since the stubs below now call the method directly rather than working through the entire callback chain.

It borders the line of testing the framework, however does cover potential regressions on the existence (and placement) of the callbacks.

Reply:
Works for me, though I might rename the test to `it "gets called on save"` since we're no longer testing whether `geom` is actually getting set here.

Reply:
Yeah makes sense. Especially because the actual testing of the `#set_geom_from_latlon` is happening below in  `geom` and `private_geom` blocks.